### DISCUSSION TITLE
POLLING_DEFAULT_INTERVAL?

### DISCUSSION THREAD

**Discussion by eried** [teslamate-org/teslamate#4159] [General]
Hi, I see a very frequent "third party location access" when driving, what are the implications of changing this values for polling?

Will this break the dashboards in grafana because they expect 15 seconds data for example?

**Comment by cwanja** [comment]
Are you referring to the icon / notification in-car?

**Reply by eried** [reply]
Yes 😬 is it possible to poll it less?

**Reply by cwanja** [reply]
Is there a reason why? The alert does no harm. Think you could turn off the streaming API and it should poll less.

**Reply by eried** [reply]
I know but since it blinks every 15 seconds it is kinda distracting and I think 5-10 mins granularity would be ok

**Reply by cwanja** [reply]
> I know but since it blinks every 15 seconds it is kinda distracting and I think 5-10 mins granularity would be ok

Think you are misunderstanding this notification. We do not control its frequency. And if you poll less, your drives and data points will be every five to 10 minutes.

**Reply by eried** [reply]
oh, so it is not blinking because teslamate? because when I open it with my phone the notification shows something else "like Owner connected" but when it is brinking every 5-15 seconds when driving says "third party access"

**Reply by cwanja** [reply]
It is definitely due to TeslaMate. However, if you change the polling in TeslaMate, your data quality will be worse. Significantly.

**Reply by eried** [reply]
Thanks! I will some days try with 

```

- POLLING_ASLEEP_INTERVAL: 300
- POLLING_CHARGING_INTERVAL: 30
- POLLING_DRIVING_INTERVAL: 300
- POLLING_ONLINE_INTERVAL: 60
- POLLING_DEFAULT_INTERVAL: 60
- POLLING_MINIMUM_INTERVAL: 30
```

I understand some quality degradations will happen, like the map will probably look wrong, but I think it will be more pleasing for me in general.

**Reply by xPudda** [reply]
Hi @eried, did you find any improvement with these values? I'm noticing excessive battery drain when teslamate is active and I would like to reduce API fetch

**Reply by eried** [reply]
oh I stopped using it :) so I never tested further

**Reply by JakobLichterfeld** [reply]
> I'm noticing excessive battery drain when teslamate is active and I would like to reduce API fetch

That is very unlikely, to not say impossible. Most likely you use a different 3rd party tool, or an iOS widget or have vehicle settings wrong (such as keep awake when 12V accessory is connected), or sentry mode or pre-conditioning or so.

**Reply by xPudda** [reply]
> > I'm noticing excessive battery drain when teslamate is active and I would like to reduce API fetch
> 
> That is very unlikely, to not say impossible. Most likely you use a different 3rd party tool, or an iOS widget or have vehicle settings wrong (such as keep awake when 12V accessory is connected), or sentry mode or pre-conditioning or so.

Hi @JakobLichterfeld although none of these settings are active I noticed that shutting down Teslamate during long trips save 10% more or less.
The drain problem is occurring during driving mode.
My tesla is a 2020 model and the battery is slightly worn.

**Reply by JakobLichterfeld** [reply]
> The drain problem is occurring during driving mode.

Impossible.
